lib/resty/upstream/healthcheck.lua

```diff
@@ -3,28 +3,33 @@ local log = ngx.log
 local ERR = ngx.ERR
 local WARN = ngx.WARN
 local DEBUG = ngx.DEBUG
+local ngx = ngx
+local error = error
+local string = string
 local sub = string.sub
 local re_find = ngx.re.find
 local new_timer = ngx.timer.at
 local shared = ngx.shared
 local debug_mode = ngx.config.debug
 local concat = table.concat
+local unpack = unpack
+if _VERSION == 'Lua 5.2' then
+    unpack = table.unpack
+end
 local tonumber = tonumber
 local tostring = tostring
 local ipairs = ipairs
 local ceil = math.ceil
 local spawn = ngx.thread.spawn
 local wait = ngx.thread.wait
 local pcall = pcall
+local setmetatable = setmetatable
 
 local _M = {
     _VERSION = '0.05'
 }
 
-if not ngx.config
-   or not ngx.config.ngx_lua_version
-   or ngx.config.ngx_lua_version < 9005
-then
+if not ngx.config or not ngx.config.ngx_lua_version or ngx.config.ngx_lua_version < 9005 then
```

Comment:
why change it?

Reply:
We should keep the original coding style.

Reply:
I will add a .lua-format file to define, what the original coding style is.

Reply:
the new line length is greater than 80.
your limit is 75

Reply:
the original coding style keep align with the parenthesis